src/index.js

```diff
@@ -1,15 +1,27 @@
-import _ from 'lodash';
 import './style.css';
 
+const displayList = document.getElementById('todolist');
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const toDoList = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: null,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: null,
+  },
+];
```

Comment:
- [ ] Kindly make sure to give each todo item an index as that can be used later to track which todo is which in your app.

Reply:
fixed! each item has its own index